x/qgb/orchestrator/client.go

```diff
@@ -0,0 +1,171 @@
+package orchestrator
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+	"os"
+	"strings"
+	"syscall"
+
+	wrapper "github.com/celestiaorg/quantum-gravity-bridge/ethereum/solidity/wrappers/QuantumGravityBridge.sol"
+	"github.com/celestiaorg/quantum-gravity-bridge/orchestrator/ethereum/keystore"
```

Comment:
Shouldn't we move this here? instead of importing ?

Reply:
good idea, https://github.com/celestiaorg/celestia-app/pull/255/commits/cc4f6009a65e1eb68b23b83032f17821346b4335